Anytype/Sources/ServiceLayer/Object/ObjectType/InstalledObjectTypes/InstalledObjectTypesProvider.swift

```diff
@@ -0,0 +1,67 @@
+import AnytypeCore
+import Services
+import ProtobufMessages
+import Combine
+
+protocol InstalledObjectTypesProviderProtocol: AnyObject {
+    var objectTypes: [ObjectType] { get }
+    var objectTypesPublisher: AnyPublisher<[ObjectType], Never> { get }
+
+    func startSubscription() async
+}
+
+final class InstalledObjectTypesProvider: InstalledObjectTypesProviderProtocol {
```

Comment:
it really needed subscription? Maybe use search request
Also we have a ObjectTypeProvider with syncPublisher. This class is very similar

Reply:
No, I can't use search request because I need to track object types from the library.
But you are right, I remake logic with ObjectTypeProvider - just filtering types after update, thanks!